packages/addons/service/tvheadend42/package.mk

```diff
@@ -91,6 +91,7 @@ post_unpack() {
 }
 
 pre_configure_target() {
+  cp -a $(get_build_dir tvh-dtv-scan-tables) $PKG_BUILD/data/dvb-scan
```

Comment:
What about making symbolic link here?

Reply:
I guess symlinking could work but after running into the issue of tvheadend build `rm -rf`-ing the files in data/dvb-scan I don't trust the tvheadend build enough to give it access to some shared resource.
see https://github.com/tvheadend/tvheadend/blob/release/4.2/Makefile#L801

Reply:
Ah, in this case you would lost complete source.
Better safe than sorry then :)

Reply:
@vpeter4 we have already an more elegant way to do it https://github.com/CvH/LibreELEC.tv/tree/9.0-tvh-scan-tables (not final just quickly done yesterday) that allows also to update the scandata from user side (addon integration is currently wip for it)